src/AzureExtension/DevBox/DevBoxInstance.cs

```diff
@@ -44,6 +45,10 @@ public enum DevBoxActionToPerform
 /// </summary>
 public class DevBoxInstance : IComputeSystem, IComputeSystem2
 {
+    [DllImport("user32.dll")]

```

Comment:
This project already has a reference to [CsWin32](https://github.com/microsoft/CsWin32) package. It would be nice to use it as we do in other projects. You'd need to add SetForegroundWindow to NativeMethods.txt and then call it as a .NET method of PInvoke class. For example, [here](https://github.com/microsoft/devhome/blob/main/HyperVExtension/src/DevSetupAgent/Requests/IsUserLoggedInRequest.cs) is a call to LsaEnumerateLogonSessions in DevHome repo.

Reply:
Looks like there is no NativeMethods.txt file or PInvoke classes being used in the solution, so that would all have to be setup.  That's beyond the scope of this change.

Reply:
Yes, you'd need to add NativeMethods.txt with one line. PInvoke class would be generated automatically. It's really simple.